checks.d/mysql.py

```diff
@@ -333,27 +673,53 @@ def _collect_system_metrics(self, host, db, tags):
             pid = self._get_server_pid(db)
 
         if pid:
-            self.log.debug("pid: %s" % pid)
-            # At last, get mysql cpu data out of procfs
+            self.log.debug("System metrics for mysql w\ pid: %s" % pid)
+            # At last, get mysql cpu data out of psutil or procfs
+
             try:
-                # See http://www.kernel.org/doc/man-pages/online/pages/man5/proc.5.html
-                # for meaning: we get 13 & 14: utime and stime, in clock ticks and convert
-                # them with the right sysconf value (SC_CLK_TCK)
-                proc_file = open("/proc/%d/stat" % pid)
-                data = proc_file.readline()
-                proc_file.close()
-                fields = data.split(' ')
-                ucpu = fields[13]
-                kcpu = fields[14]
-                clk_tck = os.sysconf(os.sysconf_names["SC_CLK_TCK"])
-
-                # Convert time to s (number of second of CPU used by mysql)
-                # It's a counter, it will be divided by the period, multiply by 100
-                # to get the percentage of CPU used by mysql over the period
-                self.rate("mysql.performance.user_time",
-                          int((float(ucpu) / float(clk_tck)) * 100), tags=tags)
-                self.rate("mysql.performance.kernel_time",
-                          int((float(kcpu) / float(clk_tck)) * 100), tags=tags)
+                proc = None
+                if PSUTIL_AVAILABLE:
```

Comment:
Let's just get rid of cpu metrics if psutil is not available. I agree it's a breaking change but it's impact is really limited.